tests/atoms.cpp

```diff
@@ -10,22 +10,29 @@ TEST_CASE("Use the Atom type", "[Atoms]"){
     Atom a2 = Atom();
     Atom a3(Atom::COARSE_GRAINED, "CH4");
     Atom a4("W");
+	Atom a5("C", "CB");
 
     SECTION("Check constructors"){
-        CHECK(a1.name() == "H");
+        CHECK(a1.element() == "H");
         CHECK(a1.mass() == 1.008f);
         CHECK(a1.type() == Atom::ELEMENT);
         CHECK(a1.charge() == 0);
 
         CHECK(a2.type() == Atom::UNDEFINED);
-        CHECK(a2.name() == "");
+        CHECK(a2.element() == "");
         CHECK(a2.mass() == 0);
         CHECK(a2.charge() == 0);
 
         CHECK(a3.type() == Atom::COARSE_GRAINED);
-        CHECK(a3.name() == "CH4");
+        CHECK(a3.element() == "CH4");
         CHECK(a3.mass() == 0);
         CHECK(a3.charge() == 0);
+
+		CHECK(a5.type() == Atom::ELEMENT);
```

Comment:
Nice to add more tests! Could you make sure the indentation is 4 spaces everywhere?